src/ol_infrastructure/applications/botkube/__main__.py

```diff
@@ -0,0 +1,673 @@
+                    "runtime": {
+                        "configMap": {
+                            "name": "botkube-runtime-config",
+                            "annotations": {},
+                        },
+                        "fileName": "_runtime_state.yaml",
+                    },
+                },
+            },
+            # SSL configuration
+            "ssl": {"enabled": False, "existingSecretName": "", "cert": ""},
+            # Service configuration
+            "service": {"name": "metrics", "port": 2112, "targetPort": 2112},
+            # ServiceMonitor configuration
+            "serviceMonitor": {
+                "enabled": False,
+                "interval": "10s",
+                "path": "/metrics",
+                "port": "metrics",
+                "labels": {},
+            },
+            # Deployment configuration
+            "deployment": {
+                "annotations": {},
+                "livenessProbe": {
+                    "initialDelaySeconds": 1,
+                    "periodSeconds": 2,
+                    "timeoutSeconds": 1,
+                    "failureThreshold": 35,
+                    "successThreshold": 1,
+                },
+                "readinessProbe": {
+                    "initialDelaySeconds": 1,
+                    "periodSeconds": 2,
+                    "timeoutSeconds": 1,
+                    "failureThreshold": 35,
+                    "successThreshold": 1,
+                },
+            },
+            # Pod configuration
+            "replicaCount": 1,
+            "extraAnnotations": {},
+            "extraLabels": {},
+            "priorityClassName": "",
+            "nameOverride": "",
+            "fullnameOverride": "",
+            # Resource limits and requests
+            "resources": {},
+            # Environment variables
+            "extraEnv": [
+                {"name": "LOG_LEVEL_SOURCE_BOTKUBE_KUBERNETES", "value": "debug"},
+                {
+                    "name": "BOTKUBE_COMMUNICATIONS_DEFAULT-GROUP_SOCKET__SLACK_APP__TOKEN",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "communication-slack",
+                            "key": "slack-bot-token",
+                        },
+                    },
+                },
+                {
+                    "name": "BOTKUBE_COMMUNICATIONS_DEFAULT-GROUP_SOCKET__SLACK_BOT__TOKEN",
+                    "valueFrom": {
+                        "secretKeyRef": {
+                            "name": "communication-slack",
+                            "key": "slack-app-token",
+                        },
+                    },
+                },
+            ],
+            # Volume configuration
+            "extraVolumes": [],
+            "extraVolumeMounts": [],
+            # Node assignment
+            "nodeSelector": {},
+            "tolerations": [],
+            "affinity": {},
+            # Service account
```

Comment:
Any of these entries that are empty can just be left out of the values to reduce visual noise.